raiden/tests/integration/test_events.py

```diff
@@ -35,9 +35,9 @@ def event_dicts_are_equal(dict1, dict2):
 
         v2 = dict2[k]
         if isinstance(v2, str) and v2.startswith('0x'):
-            v2 = v2[2:]
+            v2 = v2[2:].lower()
```

Comment:
When we switch to using EIP55 everywhere this should go away, right?

Reply:
exactly. If `address_encoder()` returns checksummed address, then the comparison will be true for values returned from `web3`.

It's also possible to compare checksummed/non-checksummed addrs using `is_same_address()`, but since the event parameters also include hashes, it would be a little more complicated.